src/test/java/org/folio/marc4ld/mapper/test/MonographTestUtil.java

```diff
@@ -164,6 +166,51 @@
 @UtilityClass
 public class MonographTestUtil {
 
+  public static Resource getSampleInstanceWithWork(Resource work) {
```

Comment:
Instead of adding custom util methods to this class, I think it is better to create this method in the IT class that need it. Else, this class will grow too big.

Reply:
fixed.